admin/lint_json.m

```diff
@@ -1,18 +1,28 @@
-function lint_json(filesin, outputfile)
+function lint_json(filesin, outputfile, varargin)
 % Use mlint, and convert to a json for easy parsing by WNG
 %
 % Input:
 % ------
-% filesin              char array OR cell array of char arrays detailing files to parse
+% filesin              cell array of char arrays detailing files to parse
 %                          if filesin is empty will recurse from current working directory
 % outputfile           char array of filename to write output to (will overwrite)
 %                          if outputfile is empty will write to stdout
+%
+% Keyword arguments:
+% exclude              cell array of char arrays detailing files to exclude from parsing
```

Comment:
What does "detailing files to" mean? Is that as regex, glob, filename, fullpaths, ...? 

(applied equally to the `filesin` parameter)